lib/connection/process.coffee

```diff
@@ -1,8 +1,32 @@
 child_process = require 'child_process'
 client = require './client'
 net = require 'net'
+path = require 'path'
+fs = require 'fs'
 
 module.exports =
+
+  bundledExe: ->
+    res = path.dirname(atom.config.resourcePath)
+    p = if process.platform is 'darwin'
+      path.join res, 'julia/bin/julia'
+    else if process.platform is 'win32'
+      path.join res, 'julia/bin/julia.exe'
```

Comment:
Do we only have OSX and Windows bundles? The linux case should be the same as OSX, no?